packages/react/README.md

```diff
@@ -6,7 +6,11 @@ Previous versions of this program were maintained at [dequelabs/react-axe](https
 
 ## React support
 
-This package does not support React 18 and above. Deque is working on alternatives for React 18+ developers and will update the community following product announcements at [axe-con 2024](https://www.deque.com/axe-con/).
+This package does not support React 18 and above. Deque released a product called [axe Developer Hub](https://www.deque.com/axe/developer-hub/). This product has [numerous JavaScript/TypeScript test framework integrations](https://docs.deque.com/developer-hub/2/en/dh-platform-support#browser-automation-platform-support) and is Deque's recommended path forward for users of this library who wish to use more modern versions of React.
```

Comment:
Does this mean that React support will disappear as an open source?

Reply:
@azu we have no plans to support React 18 and above within this library.

Reply:
I see. Thanks!